### DISCUSSION TITLE
Can't drag the Windows in Winamp Classic Interface after Ubuntu 24.04LTS upgrade

### DISCUSSION THREAD

**Discussion by carrizoja** [audacious-media-player/audacious#1394] [Q&A]
Hey! I used to use Audacious with Winamp Classic Interface. Everything worked well in Ubuntu 23.10 but, recently, I have just upgraded to Ubuntu 24.04LTS and I noticed that I couldn't drag the main windows. It remains at the top left. 
I changed to the GTK Interface and it works well. Anyone with the same problem? Regards!

![Screenshot from 2024-05-26 12-12-32](https://github.com/audacious-media-player/audacious/assets/58268794/56a0b5cd-f10e-41af-9076-f8919a68eb16)

**Comment by radioactiveman** [comment]
Yes, this has been reported several times already. It happens because Ubuntu uses Wayland by default now.

Please read https://github.com/orgs/audacious-media-player/discussions/105 and https://github.com/orgs/audacious-media-player/discussions/1392.

**Reply by carrizoja** [reply]
Thanks for your reply!
